core/src/trezor/workflow.py

```diff
@@ -26,23 +26,23 @@
 default_constructor = None  # type: Optional[Callable[[], loop.Task]]
 
 
-def on_start(workflow: loop.Task) -> None:
+def _on_start(workflow: loop.spawn) -> None:
     """
-    Call after creating a workflow task, but before running it.  You should
+    Called after creating a workflow task, but before running it.  You should
     make sure to always call `on_close` when the task is finished.
```

Comment:
Seems to me this comment is no longer relevant, since calling `_on_close()` is automated.

Reply:
Although there is an `on_close()` method (without the initial underscore) elsewhere, but as far as I can tell, that's not what is meant here.

Reply:
done in 62552006a2cacbd677bcf341163566da7544c03f